components/Alert/styles.js

```diff
@@ -9,6 +9,7 @@ const Message = styled('div')`
   background-color: ${({ status }) => statusColor[status].bg};
   border-radius: 0.25rem;
   color: ${({ status }) => statusColor[status].color};
+  font-family: Inter;
```

Comment:
```suggestion
  font-family: 'Inter', sans-serif;
```
Siempre es buena idea hacer fallback al tipo base, serif o sans-serif, en este caso pienso que es un sans-serif.

Reply:
Sí está mejor :+1: